### ISSUE TITLE
Adding countries to existing setup

### ISSUE THREAD

**Issue by umar18** [pelias/docker#180]
Hi,

I have pelias docker setup up and running for north America and its working flawlessly. Now i want to add another country. How can i do it ? Do i need to start from the scratch ? It will take too much time. Please suggest.

**Comment by wkulesza** [comment]
I was searching for a tutorial/howto add new project to projects folder, forked the code to add my region, but when i run the sh script, it goes through to the end, but i notice "elasticsearch did not come up" in the process.
Could someone verify my "wlkp" region in the projects section ?

https://github.com/wkulesza/docker

**Comment by missinglink** [comment]
The `importPlace` ID in of your `pelias.json` refers to Singapore.

You need to change it to https://spelunker.whosonfirst.org/id/85633723/

`pelias elastic start` and `pelias elastic wait` are essential, you must have a running elastic search instance before continuing.

If it's saying that it didn't come up then you need to fix that before continuing.

**Comment by wkulesza** [comment]
Hi,
thanks for this @missinglink  I have updated my fork with correct id for Poland.

See my log fragment about elasticsearch service did not come up, check configuration:

http://prntscr.com/uhjd4o

Any idea why this is happening? I see a note about outdated elasticsearch version - but im using most recent docker pelias code ?

**Comment by missinglink** [comment]
@wkulesza did you follow the instructions at https://github.com/pelias/docker
Have you successfully build the small `portland-metro` project before trying to do a custom build?

**Comment by wkulesza** [comment]
@missinglink Yes, I did, altough i had the same issue, i was able to send requests to api and receive responses.

**Comment by missinglink** [comment]
Please open a new issue and complete the questions in the issue template.
It's impossible for me to answer "it's not working, why?" without more information.
